orcid_hub/templates/section.html

```diff
@@ -45,8 +45,8 @@
             <th class="column-header">Visibility</th>
           {% else %}
             <th class="column-header">
-              {% if section_type == "EDU" %}Institution
-              {% elif section_type == "EMP" %}Institution/Employer
+              {% if section_type in ["EDU", "QUA"] %}Institution
```

Comment:
another improvement:
- instead of "Cancel" user "Close" with ".btn-default" instead of having it red 

![list_cancel](https://user-images.githubusercontent.com/177266/60229468-1d214680-98ea-11e9-8f55-11910e3be6c4.png)